src/driver_lirc.cpp

```diff
@@ -2,6 +2,7 @@
  * Copyright 2016 <Admobilize>
  * MATRIX Labs  [http://creator.matrix.one]
  * This file is part of MATRIX Creator MALOS
+ * Author: @hpsaturn
```

Comment:
Are we using the Author thing? I think we leave ADM and that's it. Not sure. You might want to ask Brian.